nl_server/gcs.py

```diff
@@ -32,5 +34,59 @@ def download_embeddings(embeddings_file: str) -> str:
   return local_embeddings_path
 
 
+def local_folder() -> str:
+  return TEMP_DIR
+
+
 def local_path(embeddings_file: str) -> str:
-  return os.path.join(TEMP_DIR, embeddings_file)
+  return os.path.join(local_folder(), embeddings_file)
+
+
+def download_model_from_gcs(gcs_bucket: Any, local_dir: str,
+                            model_folder_name: str) -> str:
+  """Downloads a Sentence Tranformer model (or finetuned version) from GCS.
+
+  Args:
+    gcs_bucket: The GCS bucket.
+    local_dir: the local folder to download to.
+    model_folder_name: the GCS bucket name for the model.
+  
+  Returns the path to the local directory where the model was downloaded to.
+  The downloaded model can then be loaded as:
+
+  ```
+      downloaded_model_path = download_model_from_gcs(bucket, dir, gcs_model_folder_name)
+      model = SentenceTransformer(downloaded_model_path)
+  ```
+  """
+  if local_dir[-1] != "/":
```

Comment:
Can we use os.path.join to avoid dealing with trailing `/`s ourselves?

Reply:
yup, done